packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -338,6 +355,17 @@ export default decorate([
                   value={host}
                 />
               </div>
+              <div className='input-group '>
```

Comment:
The inputs are stuck together, you can add a `form-group` class to add some space around them.